test/run_test

```diff
@@ -648,6 +648,8 @@ run_upgrade_test ()
         prev=$act
     done
     test "$prev" != none
+    # Check if the previous image is available in the registry
+    docker pull "$(get_image_id "$prev:remote")" || return 0
```

Comment:
If the previous image does not exist, then I would prefer to return 1, which could indicate a problem and we can track it in CI logs.

Just an improvement.

Reply:
>If the previous image does not exist, then I would prefer to return 1 ...

That is what is happening now. Which is a problem as the tests fail on architectures where previous image versions are unavailable. See #300